nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/test/java/org/nuxeo/ecm/platform/audit/AbstractAuditStorageTest.java

```diff
@@ -131,16 +131,26 @@ public void testSaveAndScroll() throws Exception {
         assertEquals(ID_FOR_AUDIT_STORAGE_TESTS, logEntry.getEventId());
     }
 
+    /**
+     *  Checks the logged events match.  Will debug the result if they don't.
+     */
+    public void checkNumberOfLoggedEvents(AbstractAuditBackend backend, AuditQueryBuilder builder) {
+        List<LogEntry> logs = backend.queryLogs(builder);
+        if (logs.size() != NUM_OF_EVENTS) {
+            System.out.println("ERROR: Incorrect log entries: "+ Arrays.toString(logs.toArray()));
+        }
+        assertEquals(NUM_OF_EVENTS, logs.size());
```

Comment:
Instead of `if` + `System.out` it's cleaner to do:

    assertEquals("Incorrect log entries: "+ Arrays.toString(logs.toArray()), NUM_OF_EVENTS, logs.size());